pkg/autopilot/controller/setup.go

```diff
@@ -140,7 +147,7 @@ func (sc *setupController) createControlNode(ctx context.Context, cf apcli.Facto
 				Name: name,
 				// Create the usual os and arch labels as this describes a controller node
 				Labels: map[string]string{
-					corev1.LabelHostname:   name,
+					corev1.LabelHostname:   nodeName,
```

Comment:
It's a bit confusing that autopilot stores the _node_ name in the _host_ name label :smile:

Reply:
Well, currently it's `ControlNode` name instead of the hostname